collection-scripts/gather

```diff
@@ -39,6 +39,10 @@ resources+=(nodenetworkstates)
 # ImageStreamTag
 resources+=(istag)
 
+# Nodes and Machines
+resources+=(nodes)
+resources+=(machines)
```

Comment:
Will machineset be needed as well ?

Reply:
we don't have them in our (planned) own image, so i think we are fine without. @beekhof WDYT?

Reply:
no requirement for machinesets from my PoV